log/log.go

```diff
@@ -55,6 +55,7 @@ type Options struct {
 	StartOffset        uint64          // byte offset in file at which to start parsing
 	FilterAdminCommand map[string]bool // admin commands to ignore
 	Debug              bool            // print trace info to STDOUT
+	RateMaxQueryTime   float32         // If Query_time > MaxQueryTime do not apply rate (do not multiply)
```

Comment:
The code comment doesn't make sense to me. It should probably mention `slow_query_log_always_write_time` since that's what it's related to.